alpaca/broker/requests.py

```diff
@@ -663,20 +663,27 @@ class OrderRequest(BaseOrderRequest):
         commission (Optional[float]): The dollar value commission you want to charge the end user.
     """
 
-    commission: Optional[float]
+    commission: Optional[float] = None
     currency: Optional[SupportedCurrencies] = None  # None = USD
 
-    @validator("type")
-    def order_type_must_be_market_for_lct(cls, value: OrderType) -> OrderType:
+    @root_validator(pre=True)
```

Comment:
@haxdds this root validator should be:
`@model_validator(mode="before")`
to fix the CI

Reply:
[fixed here](https://github.com/alpacahq/alpaca-py/pull/321)